projects/core/src/user/facade/user-consent.service.ts

```diff
@@ -11,6 +11,7 @@ import {
 } from 'rxjs/operators';
 import { AuthService } from '../../auth/facade/auth.service';
 import { Consent, ConsentTemplate } from '../../model/consent.model';
+import { OCC_USER_ID_CURRENT } from '../../occ';
```

Comment:
not sure, but often this is causing issues with ngpackagr build. No need to load from the barrel file?

Reply:
Good catch overall. 
I agree this has cause issues in the past and the fix is so small it's not worth taking the risk.  See this comment for details on the fix:  https://github.com/SAP/cloud-commerce-spartacus-storefront/pull/5654#issuecomment-561870734